pyalgostrategypool/macd_crossover_v2.py

```diff
@@ -0,0 +1,67 @@
+import talib
+from pyalgotrading.constants import *
+from pyalgotrading.strategy import StrategyBase
```

Comment:
remove all imports, run this strategy from the python build website, clear all issues using the execution, and paste the changed code here